tools/cvm-image-rewriter/plugins/04-user-authkey/README.md

```diff
@@ -0,0 +1,14 @@
+# User authkey
+
+This plugin is used to set the SSH login user and public key using environment variables ${CVM_USER} and ${CVM_AUTH_KEY}.
```

Comment:
I think it is worth mentioning this won't run unless those two environmental variables are set.

Reply:
Because `CVM_USER` has default values, while `CVM_AUTH_KEY` does not, I have clarified in the next line that if the value for `CVM_AUTH_KEY` is not set, this plugin will be skipped.